taxi/views.py

```diff
@@ -64,7 +65,7 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
+    form_class = CarForm
```

Comment:
Ensure that the `CarForm` uses checkboxes for assigning drivers to a car, as specified in the task description: 'On car creation page switch to checkboxes widget for assigning drivers to car.' This is correctly implemented in the `CarForm` class in forms.py.